recipe-server/client/control/state/recipes/actions.js

```diff
@@ -23,11 +23,11 @@ function fetchRecipeSuccess(recipe) {
 export function fetchRecipe(pk) {
   const requestId = `fetch-${pk}`;
   return makeApiRequest(requestId, `recipe/${pk}/`, { method: 'GET' })
-    .then(recipe => fetchRecipeSuccess(recipe));
+    .then(recipe => receiveRecipe(recipe));
```

Comment:
All the single-argument handlers like this can be rewritten to just `.then(whateverHandlerFunc)`